README.md

```diff
@@ -84,17 +83,17 @@ import UniversalEvents from 'fusion-plugin-universal-events';
 
 The plugin. Provides the [service API](#service-api). Typically should be registered to `UniversalEventsToken`.
 
-##### `UniversalEventsToken`
+#### Dependencies
+
+##### `FetchToken`
 
 ```js
-import {UniversalEventsToken} from 'fusion-plugin-universal-events';
```

Comment:
Isn't this token still required?

Reply:
Or not, because we just register this module as the token?

Reply:
Yes, and it's already being addressed at the `Registration API` section